timers.js

```diff
@@ -1,4 +1,35 @@
+var pauseButton = document.getElementById("pause");
+var resetButton = document.getElementById("reset");
+var startButton = document.getElementById("start");
+var seconds = 0;
+var timerId = 0;
+/* START Button*/
+startButton.addEventListener ("click", function (){
```

Comment:
this is nit-picky, but common style is to not put a space between a method name and the parenthesis calling it:

```js
// instead of this:
car.drive ("100mph");
// do this:
car.drive("100mph");
```